src/app/api/getDelegationPoints.ts

```diff
@@ -0,0 +1,49 @@
+import { encode } from "url-safe-base64";
+
+import { Pagination } from "../types/api";
+
+import { apiWrapper } from "./apiWrapper";
+
+export interface DelegationPoint {
+  staking_tx_hash_hex: string;
+  last_updated_btc_height: number;
+  staker: {
+    pk: string;
+    polkadot: number;
```

Comment:
We should not use `polkadot` in the API, this is a name reserved for a different crypto. We should use `points` everywhere cc @jrwbabylonlab